my.new.project/AmpegManualTestProject.txt

```diff
@@ -0,0 +1,80 @@
+Title: Carousel-test-01
+Summary: Verify carousel cycles through images as expected
+Pre-conditions: N/A
+Steps: 1. Be on homepage
+2. Watch carousel go through slides automatically
+Expected result: Carousel cycles through 5 images in order
+Post condition: N/A
+Actual result: Cycles through images as expected
+Status: pass
```

Comment:
No execution results should be written in the manual test case. 
We'll cover the reason later.